frontend/src/app/person-income-edit/person-income-edit.component.html

```diff
@@ -0,0 +1,28 @@
+<h1 class="mb-5" id="fullName">
+  Créer un nouveau revenu pour {{ person.firstName }} {{ person.lastName }} <ng-container *ngIf="person.nickName">({{ person.nickName }})</ng-container>
+</h1>
+
+<form (ngSubmit)="save()" #incomeForm="ngForm">
+  <div class="form-group row">
+    <label for="source" class="col-sm-3 col-form-label">Source du revenu</label>
+    <div class="col-sm-9">
+      <select class="form-control" id="source" name="source" [(ngModel)]="income.source" required>
+        <option value=""></option>
+        <option *ngFor="let source of incomeSources" [ngValue]="source">{{ source.name }}</option>
+      </select>
+    </div>
+  </div>
+  <div class="form-group row">
+    <label for="monthlyAmount" class="col-sm-3 col-form-label">Montant mensuel</label>
+    <div class="col-sm-9">
+      <input type="number" class="form-control" id="monthlyAmount" name="monthlyAmount" min="1" [max]="income.source?.maxMonthlyAmount"
+             [(ngModel)]="income.monthlyAmount"/>
+    </div>
```

Comment:
There will be an error from the backend if that's the case (that will be displayed once the errors PR is merged). 

On the frontend, once angular 5 is out, we should have min/max validators on the input that should automatically work and prevent the form to be submitted if the entered value is too large. An error message would indeed be nice to inform the user of the reason that the form can't be submitted, but I can't do that until NG 5 is out.

Not sure if that's what we should do (maybe a bigger value is acceptable, and we should only display a warning, or ask confirmation)